README.md

```diff
@@ -5,11 +5,64 @@
 
 
 - [resilience-for-software](#resilience-for-software)
+  - [What's the return on investing time in learning about resilience engineering?](#whats-the-return-on-investing-time-in-learning-about-resilience-engineering)
+  - [Where do I start? What is resilience engineering?](#where-do-i-start-what-is-resilience-engineering)
+  - [What is the relationship between resilience engineering and DevOps/SRE?](#what-is-the-relationship-between-resilience-engineering-and-devopssre)
+  - [I’m intrigued, what’s next?](#im-intrigued-whats-next)
+  - [What should I do if I have other questions?](#what-should-i-do-if-i-have-other-questions)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 <!-- prettier-ignore-end -->
 
 # resilience-for-software
 
-Introduction to resilience engineering concepts for software engineers
+The intent of this document is to provide a lightweight introduction to and motivate
+further engagement with resilience engineering through a “narrative FAQ”.
+
+## What's the return on investing time in learning about resilience engineering?
+
+If failures of complex systems (e.g., incidents) are substantially impacting the
+sustainability of your systems, the happiness of your engineers, your ability to meet
+business needs, and/or the happiness of your customers
+([framing borrowed from Honeycomb](https://www.honeycomb.io/framework-for-an-observability-maturity-model-using-observability-to-advance-your-engineering-product/)),
+resilience engineering may offer insight into which approaches to address these issues
+will be effective and high leverage.
+
+## Where do I start? What is resilience engineering?
+
+For definitions and to gain a basic familiarity with the domain, we recommend you start
+by reading [How Complex Systems Fail](https://how.complexsystems.fail) by Richard Cook,
+and then watch
+[Resilience Engineering: The What and How](https://www.youtube.com/watch?v=9f4-Z8Tasa8)
+by John Allspaw.
+
+## What is the relationship between resilience engineering and DevOps/SRE?
+
+This is a complex question without a lightweight answer. We recommend reading the
+introduction and conclusion of [Accelerate](https://itrevolution.com/book/accelerate/)
```

Comment:
does introduction mean the preface? there isn't anything labeled "introduction", i think. does conclusion mean the last chapter, titled "conclusion"?

Reply:
yes, thanks, will tweak